### PR TITLE
add role to the agent

### PR DESCRIPTION
**Pull request by ruanrongman**
This is closed to #63 ,it can provide perfect  support for The following scenarios
user: 我感觉好冷
agent: 是否需要我为你打开取暖器
user: 是的

### CODE REVIEW

**Comment by Undertone0809** [comment]
How does AutoAgent use prefix system prompt? Can you place relevant code link ?

**Comment by ruanrongman** [comment]
> How does AutoAgent use prefix system prompt? Can you place relevant code link ?

https://github.com/LinkSoul-AI/AutoAgents/blob/main/autoagents/roles/role.py

File: promptulate/agents/tool_agent/agent.py [hunk 1]
```diff
@@ -24,8 +24,14 @@ def __init__(
         tools: List[Union[BaseTool, Tool]],
         llm: BaseLLM = None,
         stop_sequences: List[str] = None,
+        prefix_prompt_template: StringTemplate = StringTemplate(PREFIX_TEMPLATE),
         system_prompt_template: StringTemplate = StringTemplate(REACT_ZERO_SHOT_PROMPT),
         hooks: List[Callable] = None,
+        enable_role: bool = False,
+        profile: str = "bot",
```
Comment (Undertone0809) [inline promptulate/agents/tool_agent/agent.py:31]: Profile and name seems similar. Would it be better to keep one of them?

Comment (Undertone0809) [reply promptulate/agents/tool_agent/agent.py:31]: profile is a  unclear concept.

Comment (ruanrongman) [reply promptulate/agents/tool_agent/agent.py:31]: I think so

Comment (Undertone0809) [reply promptulate/agents/tool_agent/agent.py:31]: I think we can change "profile" to "agent_identity".

Comment (Undertone0809) [reply promptulate/agents/tool_agent/agent.py:31]: I think we can change "profile" to "agent_identity".

**Comment by Undertone0809** [comment]
Can you place relevant docs？

**Pull request merged**